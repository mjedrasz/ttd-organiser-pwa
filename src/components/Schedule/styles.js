import { makeStyles } from '@material-ui/core/styles';

const useStyles = makeStyles(theme => ({
  paper: {
    marginTop: theme.spacing(8),
    display: 'flex',
    flexDirection: 'column',
    alignItems: 'center',
  },
  avatar: {
    margin: theme.spacing(1),
    backgroundColor: theme.palette.secondary.main,
  },
  form: {
    width: '100%', // Fix IE 11 issue.
    marginTop: theme.spacing(3),
  },
  submit: {
    margin: theme.spacing(3, 0, 2),
  },
  wrapper: {
    position: 'relative',
  },
  submitProgress: {
    color: theme.palette.secondary.main,
    position: 'absolute',
    top: '50%',
    left: '50%',
    marginTop: -8,
    marginLeft: -12,
  },
  deleteButton: {
    top: theme.spacing(1),
    position: 'absolute',
    right: 0
  },
  toggleContainer: {
    margin: theme.spacing(2, 0),
  },
  toggleButton: {
    width: 50,
    [theme.breakpoints.down('xs')]: {
      width: 33,
    },
  },
  divider: {
    width: '100%',
    marginBottom: theme.spacing(3),
  },
  weekDays: {
    width: '100%',
    justifyContent: 'space-between'
  },
}));

export default useStyles;