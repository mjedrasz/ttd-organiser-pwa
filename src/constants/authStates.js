const SIGNED_IN = 'signedIn';
const SIGN_IN = 'signIn';
const CONFIRM_SIGN_UP = 'confirmSignUp';
const SIGN_UP = 'signUp';
const FORGOT_PASSWORD = 'forgotPassword';
const RESET_PASSWORD = 'resetPassword';
const VERIFY_CONTACT = 'verifyContact';

export {
    SIGNED_IN,
    SIGN_IN,
    CONFIRM_SIGN_UP, 
    SIGN_UP,
    FORGOT_PASSWORD,
    RESET_PASSWORD,
    VERIFY_CONTACT
};