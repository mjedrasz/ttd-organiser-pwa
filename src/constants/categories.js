import { I18n } from "aws-amplify";

const MUSIC = "MUSIC";
const DANCE = "DANCE";
const ARTS = "ARTS";
const CRAFTS = "CRAFTS";
const SPORT = "SPORT";
const CULTURE = "CULTURE";
const OUTDOOR = "OUTDOOR";
const CINEMA = "CINEMA";
const THEATER = "THEATER";
const PERSONAL_DEVELOPMENT = "PERSONAL_DEVELOPMENT";
const PARTY = "PARTY";
const OTHER = "OTHER";

const labels = {
    [MUSIC]: I18n.get("Music"),
    [DANCE]: I18n.get("Dance"),
    [ARTS]: I18n.get("Arts"),
    [CRAFTS]: I18n.get("Crafts"),
    [SPORT]: I18n.get("Sport"),
    [OUTDOOR]: I18n.get("Outdoor"),
    [CULTURE]: I18n.get("Culture"),
    [CINEMA]: I18n.get("Cinema"),
    [THEATER]: I18n.get("Theater"),
    [PERSONAL_DEVELOPMENT]: I18n.get("Personal development"),
    [PARTY]: I18n.get("Party"),
    [OTHER]: I18n.get("Other")
};
const keys = [
    MUSIC,
    DANCE,
    ARTS,
    CRAFTS,
    SPORT,
    OUTDOOR,
    CULTURE,
    CINEMA,
    THEATER,
    PERSONAL_DEVELOPMENT,
    PARTY,
    OTHER];
    
export {
    keys,
    labels,
    MUSIC,
    DANCE,
    ARTS,
    CRAFTS,
    SPORT,
    OUTDOOR,
    CULTURE,
    CINEMA,
    THEATER,
    PERSONAL_DEVELOPMENT,
    PARTY,
    OTHER
};