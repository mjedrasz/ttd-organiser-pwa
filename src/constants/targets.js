import { I18n } from "aws-amplify";

const ADULT = "ADULT";
const FAMILY = "FAMILY";
const CHILD = "CHILD";

const labels = {
    [ADULT]: I18n.get("Adult"),
    [FAMILY]: I18n.get("Family"),
    [CHILD]: I18n.get("Child")
};

const keys = [
    ADULT,
    FAMILY,
    CHILD
];
    
export {
    keys,
    labels,
    ADULT,
    FAMILY,
    CHILD
};