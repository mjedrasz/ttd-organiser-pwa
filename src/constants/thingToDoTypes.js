import { I18n } from "aws-amplify";

const EVENT = "EVENT";
const RECURRING_EVENT = "RECURRING_EVENT";
const PLACE = "PLACE";

const labels = {
    [EVENT]: I18n.get("Event"),
    [RECURRING_EVENT]: I18n.get("Recurring event"),
    [PLACE]: I18n.get("Place")
};

const keys = [
    EVENT,
    RECURRING_EVENT,
    PLACE
];
    
export {
    keys,
    labels,
    EVENT,
    RECURRING_EVENT,
    PLACE
};