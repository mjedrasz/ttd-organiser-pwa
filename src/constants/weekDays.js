import { I18n } from "aws-amplify";

const MON = "MON";
const TUE = "TUE";
const WED = "WED";
const THU = "THU";
const FRI = "FRI";
const SAT = "SAT";
const SUN = "SUN";

const labels = {
    [MON]: I18n.get("Mon"),
    [TUE]: I18n.get("Tue"),
    [WED]: I18n.get("Wed"),
    [THU]: I18n.get("Thu"),
    [FRI]: I18n.get("Fri"),
    [SAT]: I18n.get("Sat"),
    [SUN]: I18n.get("Sun")
};

const keys = [
    MON,
    TUE,
    WED,
    THU,
    FRI,
    SAT,
    SUN
];
    
export {
    keys,
    labels,
    MON,
    TUE,
    WED,
    THU,
    FRI,
    SAT,
    SUN
};