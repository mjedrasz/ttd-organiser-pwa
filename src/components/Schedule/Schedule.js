import React from 'react';
import Grid from '@material-ui/core/Grid';
import Divider from '@material-ui/core/Divider';
import Typography from '@material-ui/core/Typography';
import Fab from '@material-ui/core/Fab';
import DeleteIcon from '@material-ui/icons/Delete';
import { Logger } from '@aws-amplify/core';
import I18n from "@aws-amplify/core/lib/I18n";
import ToggleButton from '@material-ui/lab/ToggleButton';
import ToggleButtonGroup from '@material-ui/lab/ToggleButtonGroup';
import * as WeekDay from '../../constants/weekDays';
import { TimePicker } from '@material-ui/pickers';
import useStyles from './styles';


export default ({ schedule, handleTimeChange, handleDaysChange, handleScheduleDelete, disabled }) => {

  const classes = useStyles();

  const weekDays = WeekDay.keys.map((day) => (
    <ToggleButton key={day} value={day} className={classes.toggleButton} disabled={disabled}>
      <Typography variant="caption">{WeekDay.labels[day]}</Typography>
    </ToggleButton>));

  return (
    <>
      {schedule.map((time, index) => (
        <div key={index} className={classes.wrapper}>
          <Grid container spacing={2}>
            <Grid item xs={8}>
              <TimePicker value={time.fromTime} onChange={handleTimeChange(index, 'fromTime')}
                inputVariant="outlined"
                label={I18n.get("From")}
                disabled={disabled} />
            </Grid>
            <Grid item xs={8}>
              <TimePicker value={time.toTime} onChange={handleTimeChange(index, 'toTime')}
                inputVariant="outlined"
                
                label={I18n.get("To")}
                disabled={disabled} />
            </Grid>
            <Grid item xs={6}>
              <ToggleButtonGroup size="small" value={time.days} onChange={handleDaysChange(index)}>
                {weekDays}
              </ToggleButtonGroup>
            </Grid>
            <Grid item xs={12}>
              <Divider className={classes.divider} />
            </Grid></Grid>
          <Fab color='secondary' size='small' className={classes.deleteButton} aria-label='add'
            onClick={handleScheduleDelete(index)}
            disabled={disabled}>
            <DeleteIcon />
          </Fab>
        </div>
      ))}
    </>
  );
};
