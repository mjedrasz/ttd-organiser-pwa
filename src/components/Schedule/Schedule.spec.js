import React from 'react';
import renderer from 'react-test-renderer';
import Schedule from './Schedule';
import * as WeekDay from '../../constants/weekDays';
import DateFnsUtils from '@date-io/date-fns';
import { MuiPickersUtilsProvider } from '@material-ui/pickers';

describe('Schedule', () => {

  test('snapshot renders correctly', () => {
    const occurrences = [{
      fromTime: new Date(2019, 10, 12, 14, 0, 0, 0),
      toTime: new Date(2019, 10, 12, 16, 0, 0, 0),
      days: [WeekDay.MON, WeekDay.WED]
    },
    {
      fromTime: new Date(2019, 10, 12, 18, 0, 0, 0),
      toTime: new Date(2019, 10, 12, 20, 0, 0, 0),
      days: [WeekDay.THU, WeekDay.FRI, WeekDay.SAT]
    }]
    const component = renderer.create(<MuiPickersUtilsProvider utils={DateFnsUtils}>
      <Schedule
        schedule={occurrences}
        handleTimeChange={() => { }}
        handleDaysChange={() => { }}
        handleScheduleDelete={() => { }}
        disabled={false} />
    </MuiPickersUtilsProvider>
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

});