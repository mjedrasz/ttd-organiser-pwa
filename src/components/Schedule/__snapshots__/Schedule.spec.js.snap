// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Schedule snapshot renders correctly 1`] = `
Array [
  <div
    className="makeStyles-wrapper-5"
  >
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2"
    >
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-8"
      >
        <div
          className="MuiFormControl-root MuiTextField-root"
          format="hh:mm a"
          onClick={[Function]}
          onKeyDown={[Function]}
        >
          <label
            className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-outlined MuiFormLabel-filled"
            data-shrink={true}
          >
            From
          </label>
          <div
            className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
            onClick={[Function]}
          >
            <input
              aria-invalid={false}
              autoFocus={false}
              className="MuiInputBase-input MuiOutlinedInput-input"
              disabled={false}
              onAnimationStart={[Function]}
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              readOnly={true}
              required={false}
              type="text"
              value="02:00 PM"
            />
            <fieldset
              aria-hidden={true}
              className="PrivateNotchedOutline-root-175 MuiOutlinedInput-notchedOutline"
              style={
                Object {
                  "paddingLeft": 8,
                }
              }
            >
              <legend
                className="PrivateNotchedOutline-legend-176"
                style={
                  Object {
                    "width": 0,
                  }
                }
              >
                <span
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "&#8203;",
                    }
                  }
                />
              </legend>
            </fieldset>
          </div>
          
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-8"
      >
        <div
          className="MuiFormControl-root MuiTextField-root"
          format="hh:mm a"
          onClick={[Function]}
          onKeyDown={[Function]}
        >
          <label
            className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-outlined MuiFormLabel-filled"
            data-shrink={true}
          >
            To
          </label>
          <div
            className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
            onClick={[Function]}
          >
            <input
              aria-invalid={false}
              autoFocus={false}
              className="MuiInputBase-input MuiOutlinedInput-input"
              disabled={false}
              onAnimationStart={[Function]}
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              readOnly={true}
              required={false}
              type="text"
              value="04:00 PM"
            />
            <fieldset
              aria-hidden={true}
              className="PrivateNotchedOutline-root-175 MuiOutlinedInput-notchedOutline"
              style={
                Object {
                  "paddingLeft": 8,
                }
              }
            >
              <legend
                className="PrivateNotchedOutline-legend-176"
                style={
                  Object {
                    "width": 0,
                  }
                }
              >
                <span
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "&#8203;",
                    }
                  }
                />
              </legend>
            </fieldset>
          </div>
          
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6"
      >
        <div
          className="MuiToggleButtonGroup-root"
          role="group"
        >
          <button
            aria-pressed={true}
            className="MuiButtonBase-root MuiToggleButton-root MuiToggleButtonGroup-grouped makeStyles-toggleButton-9 MuiToggleButtonGroup-groupedSizeSmall Mui-selected MuiToggleButton-sizeSmall"
            disabled={false}
            onBlur={[Function]}
            onChange={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={0}
            type="button"
            value="MON"
          >
            <span
              className="MuiToggleButton-label"
            >
              <span
                className="MuiTypography-root MuiTypography-caption"
              >
                Mon
              </span>
            </span>
          </button>
          <button
            aria-pressed={false}
            className="MuiButtonBase-root MuiToggleButton-root MuiToggleButtonGroup-grouped makeStyles-toggleButton-9 MuiToggleButtonGroup-groupedSizeSmall MuiToggleButton-sizeSmall"
            disabled={false}
            onBlur={[Function]}
            onChange={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={0}
            type="button"
            value="TUE"
          >
            <span
              className="MuiToggleButton-label"
            >
              <span
                className="MuiTypography-root MuiTypography-caption"
              >
                Tue
              </span>
            </span>
          </button>
          <button
            aria-pressed={true}
            className="MuiButtonBase-root MuiToggleButton-root MuiToggleButtonGroup-grouped makeStyles-toggleButton-9 MuiToggleButtonGroup-groupedSizeSmall Mui-selected MuiToggleButton-sizeSmall"
            disabled={false}
            onBlur={[Function]}
            onChange={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={0}
            type="button"
            value="WED"
          >
            <span
              className="MuiToggleButton-label"
            >
              <span
                className="MuiTypography-root MuiTypography-caption"
              >
                Wed
              </span>
            </span>
          </button>
          <button
            aria-pressed={false}
            className="MuiButtonBase-root MuiToggleButton-root MuiToggleButtonGroup-grouped makeStyles-toggleButton-9 MuiToggleButtonGroup-groupedSizeSmall MuiToggleButton-sizeSmall"
            disabled={false}
            onBlur={[Function]}
            onChange={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={0}
            type="button"
            value="THU"
          >
            <span
              className="MuiToggleButton-label"
            >
              <span
                className="MuiTypography-root MuiTypography-caption"
              >
                Thu
              </span>
            </span>
          </button>
          <button
            aria-pressed={false}
            className="MuiButtonBase-root MuiToggleButton-root MuiToggleButtonGroup-grouped makeStyles-toggleButton-9 MuiToggleButtonGroup-groupedSizeSmall MuiToggleButton-sizeSmall"
            disabled={false}
            onBlur={[Function]}
            onChange={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={0}
            type="button"
            value="FRI"
          >
            <span
              className="MuiToggleButton-label"
            >
              <span
                className="MuiTypography-root MuiTypography-caption"
              >
                Fri
              </span>
            </span>
          </button>
          <button
            aria-pressed={false}
            className="MuiButtonBase-root MuiToggleButton-root MuiToggleButtonGroup-grouped makeStyles-toggleButton-9 MuiToggleButtonGroup-groupedSizeSmall MuiToggleButton-sizeSmall"
            disabled={false}
            onBlur={[Function]}
            onChange={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={0}
            type="button"
            value="SAT"
          >
            <span
              className="MuiToggleButton-label"
            >
              <span
                className="MuiTypography-root MuiTypography-caption"
              >
                Sat
              </span>
            </span>
          </button>
          <button
            aria-pressed={false}
            className="MuiButtonBase-root MuiToggleButton-root MuiToggleButtonGroup-grouped makeStyles-toggleButton-9 MuiToggleButtonGroup-groupedSizeSmall MuiToggleButton-sizeSmall"
            disabled={false}
            onBlur={[Function]}
            onChange={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={0}
            type="button"
            value="SUN"
          >
            <span
              className="MuiToggleButton-label"
            >
              <span
                className="MuiTypography-root MuiTypography-caption"
              >
                Sun
              </span>
            </span>
          </button>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
      >
        <hr
          className="MuiDivider-root makeStyles-divider-10"
        />
      </div>
    </div>
    <button
      aria-label="add"
      className="MuiButtonBase-root MuiFab-root makeStyles-deleteButton-7 MuiFab-sizeSmall MuiFab-secondary"
      disabled={false}
      onBlur={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiFab-label"
      >
        <svg
          aria-hidden="true"
          className="MuiSvgIcon-root"
          focusable="false"
          role="presentation"
          viewBox="0 0 24 24"
        >
          <path
            d="M6 19c0 1.1.9 2 2 2h8c1.1 0 2-.9 2-2V7H6v12zM19 4h-3.5l-1-1h-5l-1 1H5v2h14V4z"
          />
        </svg>
      </span>
    </button>
  </div>,
  <div
    className="makeStyles-wrapper-5"
  >
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2"
    >
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-8"
      >
        <div
          className="MuiFormControl-root MuiTextField-root"
          format="hh:mm a"
          onClick={[Function]}
          onKeyDown={[Function]}
        >
          <label
            className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-outlined MuiFormLabel-filled"
            data-shrink={true}
          >
            From
          </label>
          <div
            className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
            onClick={[Function]}
          >
            <input
              aria-invalid={false}
              autoFocus={false}
              className="MuiInputBase-input MuiOutlinedInput-input"
              disabled={false}
              onAnimationStart={[Function]}
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              readOnly={true}
              required={false}
              type="text"
              value="06:00 PM"
            />
            <fieldset
              aria-hidden={true}
              className="PrivateNotchedOutline-root-175 MuiOutlinedInput-notchedOutline"
              style={
                Object {
                  "paddingLeft": 8,
                }
              }
            >
              <legend
                className="PrivateNotchedOutline-legend-176"
                style={
                  Object {
                    "width": 0,
                  }
                }
              >
                <span
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "&#8203;",
                    }
                  }
                />
              </legend>
            </fieldset>
          </div>
          
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-8"
      >
        <div
          className="MuiFormControl-root MuiTextField-root"
          format="hh:mm a"
          onClick={[Function]}
          onKeyDown={[Function]}
        >
          <label
            className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-outlined MuiFormLabel-filled"
            data-shrink={true}
          >
            To
          </label>
          <div
            className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
            onClick={[Function]}
          >
            <input
              aria-invalid={false}
              autoFocus={false}
              className="MuiInputBase-input MuiOutlinedInput-input"
              disabled={false}
              onAnimationStart={[Function]}
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              readOnly={true}
              required={false}
              type="text"
              value="08:00 PM"
            />
            <fieldset
              aria-hidden={true}
              className="PrivateNotchedOutline-root-175 MuiOutlinedInput-notchedOutline"
              style={
                Object {
                  "paddingLeft": 8,
                }
              }
            >
              <legend
                className="PrivateNotchedOutline-legend-176"
                style={
                  Object {
                    "width": 0,
                  }
                }
              >
                <span
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "&#8203;",
                    }
                  }
                />
              </legend>
            </fieldset>
          </div>
          
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6"
      >
        <div
          className="MuiToggleButtonGroup-root"
          role="group"
        >
          <button
            aria-pressed={false}
            className="MuiButtonBase-root MuiToggleButton-root MuiToggleButtonGroup-grouped makeStyles-toggleButton-9 MuiToggleButtonGroup-groupedSizeSmall MuiToggleButton-sizeSmall"
            disabled={false}
            onBlur={[Function]}
            onChange={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={0}
            type="button"
            value="MON"
          >
            <span
              className="MuiToggleButton-label"
            >
              <span
                className="MuiTypography-root MuiTypography-caption"
              >
                Mon
              </span>
            </span>
          </button>
          <button
            aria-pressed={false}
            className="MuiButtonBase-root MuiToggleButton-root MuiToggleButtonGroup-grouped makeStyles-toggleButton-9 MuiToggleButtonGroup-groupedSizeSmall MuiToggleButton-sizeSmall"
            disabled={false}
            onBlur={[Function]}
            onChange={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={0}
            type="button"
            value="TUE"
          >
            <span
              className="MuiToggleButton-label"
            >
              <span
                className="MuiTypography-root MuiTypography-caption"
              >
                Tue
              </span>
            </span>
          </button>
          <button
            aria-pressed={false}
            className="MuiButtonBase-root MuiToggleButton-root MuiToggleButtonGroup-grouped makeStyles-toggleButton-9 MuiToggleButtonGroup-groupedSizeSmall MuiToggleButton-sizeSmall"
            disabled={false}
            onBlur={[Function]}
            onChange={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={0}
            type="button"
            value="WED"
          >
            <span
              className="MuiToggleButton-label"
            >
              <span
                className="MuiTypography-root MuiTypography-caption"
              >
                Wed
              </span>
            </span>
          </button>
          <button
            aria-pressed={true}
            className="MuiButtonBase-root MuiToggleButton-root MuiToggleButtonGroup-grouped makeStyles-toggleButton-9 MuiToggleButtonGroup-groupedSizeSmall Mui-selected MuiToggleButton-sizeSmall"
            disabled={false}
            onBlur={[Function]}
            onChange={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={0}
            type="button"
            value="THU"
          >
            <span
              className="MuiToggleButton-label"
            >
              <span
                className="MuiTypography-root MuiTypography-caption"
              >
                Thu
              </span>
            </span>
          </button>
          <button
            aria-pressed={true}
            className="MuiButtonBase-root MuiToggleButton-root MuiToggleButtonGroup-grouped makeStyles-toggleButton-9 MuiToggleButtonGroup-groupedSizeSmall Mui-selected MuiToggleButton-sizeSmall"
            disabled={false}
            onBlur={[Function]}
            onChange={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={0}
            type="button"
            value="FRI"
          >
            <span
              className="MuiToggleButton-label"
            >
              <span
                className="MuiTypography-root MuiTypography-caption"
              >
                Fri
              </span>
            </span>
          </button>
          <button
            aria-pressed={true}
            className="MuiButtonBase-root MuiToggleButton-root MuiToggleButtonGroup-grouped makeStyles-toggleButton-9 MuiToggleButtonGroup-groupedSizeSmall Mui-selected MuiToggleButton-sizeSmall"
            disabled={false}
            onBlur={[Function]}
            onChange={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={0}
            type="button"
            value="SAT"
          >
            <span
              className="MuiToggleButton-label"
            >
              <span
                className="MuiTypography-root MuiTypography-caption"
              >
                Sat
              </span>
            </span>
          </button>
          <button
            aria-pressed={false}
            className="MuiButtonBase-root MuiToggleButton-root MuiToggleButtonGroup-grouped makeStyles-toggleButton-9 MuiToggleButtonGroup-groupedSizeSmall MuiToggleButton-sizeSmall"
            disabled={false}
            onBlur={[Function]}
            onChange={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={0}
            type="button"
            value="SUN"
          >
            <span
              className="MuiToggleButton-label"
            >
              <span
                className="MuiTypography-root MuiTypography-caption"
              >
                Sun
              </span>
            </span>
          </button>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
      >
        <hr
          className="MuiDivider-root makeStyles-divider-10"
        />
      </div>
    </div>
    <button
      aria-label="add"
      className="MuiButtonBase-root MuiFab-root makeStyles-deleteButton-7 MuiFab-sizeSmall MuiFab-secondary"
      disabled={false}
      onBlur={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiFab-label"
      >
        <svg
          aria-hidden="true"
          className="MuiSvgIcon-root"
          focusable="false"
          role="presentation"
          viewBox="0 0 24 24"
        >
          <path
            d="M6 19c0 1.1.9 2 2 2h8c1.1 0 2-.9 2-2V7H6v12zM19 4h-3.5l-1-1h-5l-1 1H5v2h14V4z"
          />
        </svg>
      </span>
    </button>
  </div>,
]
`;
